thread/src/components/AddContactForm.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
```

Comment:
This whole component is in the wrong location, should be in `components/Contacts/AddContactForm.js`.
I think pulling from `go-live-branding` will fix that